addons/dark-www/addon.json

```diff
@@ -2297,6 +2297,25 @@
       "name": "\"Experimental\" Dark",
       "id": "experimentalDark",
       "description": "A dark theme by Maximouse based on a userstyle. Not experimental",
+      "values": {
+        "page": "#202020",
+        "navbar": "#855cd6",
+        "box": "#282828",
+        "gray": "#333333",
+        "blue": "#292d32",
```

Comment:
The default value of this setting needs to be updated to match the Experimental Dark preset.

Reply:
Good catch, the addon defaults should match the `"Experimental" Dark` preset!